src/pages/networks/forms/NetworkForm.tsx

```diff
@@ -0,0 +1,732 @@
+      ["ipv6.address"]: values.ipv6_address,
+      ["ipv6.dhcp"]: values.ipv6_dhcp?.toString(),
+      ["ipv6.dhcp.expiry"]: values.ipv6_dhcp_expiry,
+      ["ipv6.dhcp.ranges"]: values.ipv6_dhcp_ranges,
+      ["ipv6.dhcp.stateful"]: values.ipv6_dhcp_stateful?.toString(),
+      ["ipv6.l3only"]: values.ipv6_l3only?.toString(),
+      ["ipv6.nat"]: values.ipv6_nat?.toString(),
+      ["ipv6.nat.address"]: values.ipv6_nat_address,
+      ["ipv6.ovn.ranges"]: values.ipv6_ovn_ranges,
+      ["network"]: values.network,
+    },
+  };
+};
+
+interface Props {
+  formik: FormikProps<NetworkFormValues>;
+  getYaml: () => string;
+  project: string;
+}
+
+const NetworkForm: FC<Props> = ({ formik, getYaml, project }) => {
+  const notify = useNotify();
+  const [section, setSection] = useState(MAIN_CONFIGURATION);
+  const { data: settings, isLoading: isSettingsLoading } = useSettings();
+  const hasOvn = Boolean(settings?.config["network.ovn.northbound_connection"]);
+  const hasFan = settings?.environment?.os_name
+    ?.toLowerCase()
+    .includes("ubuntu");
+
+  const { data: networks = [], isLoading: isNetworkLoading } = useQuery({
+    queryKey: [queryKeys.networks],
+    queryFn: () => fetchNetworks(project),
+  });
+
+  const updateFormHeight = () => {
+    updateMaxHeight("form-contents", "p-bottom-controls");
+  };
+  useEffect(updateFormHeight, [notify.notification?.message, section]);
+  useEventListener("resize", updateFormHeight);
+
+  if (isSettingsLoading || isNetworkLoading) {
+    return <Loader />;
+  }
+
+  const getNetworkOptions = () => {
+    const options = networks.map((network) => {
+      return {
+        label: network.name,
+        value: network.name,
+      };
+    });
+    options.unshift({
+      label: networks.length === 0 ? "No networks available" : "Select option",
+      value: "",
+    });
+    return options;
+  };
+
+  const getFormProps = (id: "type" | "network" | "name" | "description") => {
+    return {
+      id: id,
+      name: id,
+      onBlur: formik.handleBlur,
+      onChange: formik.handleChange,
+      value: formik.values[id] ?? "",
+      error: formik.touched[id] ? (formik.errors[id] as ReactNode) : null,
+      placeholder: `Enter ${id.replaceAll("_", " ")}`,
+    };
+  };
+
+  return (
+    <Form
+      className="form network-form"
+      onSubmit={() => void formik.submitForm()}
+    >
+      <NetworkFormMenu
+        active={section}
+        setActive={setSection}
+        formik={formik}
+      />
+      <Row className="form-contents" key={section}>
+        <Col size={12}>
+          {section === MAIN_CONFIGURATION && (
+            <React.Fragment>
+              <Row>
+                <Col size={8}>
+                  <Select
+                    {...getFormProps("type")}
+                    label="Type"
+                    help="Bridge (FAN) is only available on ubuntu, OVN needs to be configured to be available, Macvlan,  SR-IOV and Physical not available in the UI"
```

Comment:
As discussed, this help text can be confusing for inexperienced users who are not on a clustered server, tricking them into thinking that they just need to configure OVN in order to get the option enabled, which is currently not the case. The help text should reflect the actual restriction in place, which is the `network.ovn.northbound_connection` config being set.
Also, maybe a "Learn more" link which goes to this [page](https://documentation.ubuntu.com/lxd/en/latest/howto/network_ovn_setup/#set-up-a-lxd-cluster-on-ovn) can be helpful here.